web/pandas/pdeps/accepted/0001-purpose-and-guidelines.md

```diff
@@ -0,0 +1,110 @@
+
+### Workflow
+
+#### Submitting a PDEP
+
+Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/accepted/`.
+The file is a markdown file, you can use `web/pdeps/accepted/0001.md` as a reference
+for the expected format.
+
+By default, we expect a PDEP will be accepted, so the PR of a PDEP should be done
+in the `accepted` directory and contain `Status: Accepted`. If a PDEP is finally
+rejected, its status and directory will be updated by the core team before merging,
+once the decision is made. Please make sure you select the option
+`Allow edits and access to secrets by maintainers` when opening the PR.
+
+#### Accepted PDEP
+
+A PDEP will be accepted by the core development team, and decisions will be made
+based on the [pandas governance document](https://github.com/pandas-dev/pandas-governance/blob/master/governance.md).
+
+Once a PDEP is accepted, any contributions can be made toward the implementing the PDEP with an open-ended completion timeline . The
+pandas project development, with a mix of volunteers and developers paid from
+different sources, and development priorities are difficult to understand or
+forecast. For companies, institutions or individuals with interest in seeing a
+PDEP being implemented, or to in general see progress to the pandas roadmap,
+please check how you can help in the [contributing page](/contribute.html).
+
+#### Implemented PDEP
+
+Once a PDEP is implemented and available in the main branch of pandas, its
+status will be changed to implemented, so there is visibility that the PDEP
+is not part of the roadmap and future plans, but a change that it already
+happened. The first pandas version in which the PDEP implementation is
+available will also be included in the PDEP.
+
+#### Rejected PDEP
+
+A PDEP can be rejected when the final decision is that its implementation is
+not the best for the interests of the project. They are as useful as accepted
+PDEPs, since there are discussions that are worth having, and decisions about
+changes to pandas being made. They will be merged with `Status: Rejected`, so
+there is visibility on what was discussed and what was the outcome of the
+discussion. A PDEP can be rejected for different reasons, for example good ideas
+that aren't backward-compatible, and the breaking changes aren't considered worth
+implementing.
+
+#### Invalid PDEP
+
+For submitted PDEPs that do not contain proper documentation, are out of scope, or
+are not useful to the community for any other reason, the PR will be closed after
+discussion with the author, instead of merging them as rejected. This is to not
+add noise to the list of rejected PDEPs, which should contain documentation as
+good as an accepted PDEP, but where the final decision was to not implement the changes.
+
+## Evolution of this PDEP
+
+While most PDEPs aren't expected to change after accepted, in some cases like this
+PDEP, they will be updated to contain its latest version, if things evolve. A log
```

Comment:
We should probably clarify when a PDEP can be revised v.s. when a new PDEP has to be submitted superseding the old PDEP.